### PR TITLE
fixed media field type

### PR DESCRIPTION
**Pull request by spektra2147**

### CODE REVIEW

**Review by fatihalp — APPROVED** [review]

File: addons/default/visiosoft/advs-module/resources/lang/en/setting.php [hunk 1]
```diff
@@ -79,21 +79,6 @@
     'default_GET' => [
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/lang/en/setting.php:79]: 'default_get'

File: addons/default/visiosoft/advs-module/resources/lang/en/setting.php [hunk 2]
```diff
@@ -158,4 +131,49 @@
     'popular_ads_limit' => [
         'name' => 'Popular Ads Limit',
     ],
+
+    //Image
+
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/lang/en/setting.php:136]: remove empty line

File: addons/default/visiosoft/media-field_type/src/Http/Controller/UploadController.php [hunk 3]
```diff
@@ -25,16 +23,22 @@
  */
 class UploadController extends AdminController
 {
+    public $uploader;
+
+    public $folders;
+
+    public $files;
```
Comment (diashalabi) [inline addons/default/visiosoft/media-field_type/src/Http/Controller/UploadController.php:30]: remove empty lines between variables

**Review by diashalabi — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/advs-module/resources/config/settings/settings.php [hunk 4]
```diff
@@ -255,4 +190,102 @@
             'default_value' => 15,
```
[the remark this answers is not in the document]
Comment (spektra2147) [reply addons/default/visiosoft/advs-module/resources/config/settings/settings.php:190]: default value ? popular_ads_limit

[the remark this answers is not in the document]
Comment (diashalabi) [reply addons/default/visiosoft/advs-module/resources/config/settings/settings.php:190]: my bad

**Review by diashalabi — APPROVED** [review]

**Pull request merged**